docs/openstack/glance_adoption.md

```diff
@@ -22,13 +22,21 @@ This guide also assumes that:
 * Previous Adoption steps completed. Notably, MariaDB and Keystone
   should be already adopted.
 
+## Variables
+
+Chose the CSI storage class to use for persistent data (here we use the default one for CRC):
+
+  ```
+  STORAGE_CLASS_NAME=crc-csi-hostpath-provisioner
+  ```
+
 ## Procedure - Glance adoption
 
 As already done for [Keystone](https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/keystone_adoption.md), the Glance Adoption follows the same pattern.
 
 ### Using local storage backend
 
-When Glance should be deployed with local storage backend (not Ceph),
+When Glance should be deployed without Ceph,
```

Comment:
This change is wrong.

Even when glance is deployed with Ceph we still need local storage for import plugins.

  https://github.com/openstack-k8s-operators/glance-operator/tree/main/config/samples/import_plugins

So Glance should always be deployed with a PVC regardless of the storage backend.